src/function/open-referral-function/src/FamilyHubs.OpenReferral.Function/FamilyHubs.OpenReferral.Function.csproj

```diff
@@ -0,0 +1,35 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+    <PropertyGroup>
+        <TargetFramework>net8.0</TargetFramework>
+        <AzureFunctionsVersion>V4</AzureFunctionsVersion>
+        <OutputType>Exe</OutputType>
+        <ImplicitUsings>enable</ImplicitUsings>
+        <Nullable>enable</Nullable>
+    </PropertyGroup>
+    <ItemGroup>
+        <FrameworkReference Include="Microsoft.AspNetCore.App"/>
+        <PackageReference Include="Microsoft.Azure.Functions.Worker" Version="1.23.0" />
+        <PackageReference Include="Microsoft.Azure.Functions.Worker.Extensions.Http" Version="3.2.0" />
+        <PackageReference Include="Microsoft.Azure.Functions.Worker.Extensions.Http.AspNetCore" Version="1.3.2" />
+        <PackageReference Include="Microsoft.Azure.Functions.Worker.Sdk" Version="1.17.4" />
+        <PackageReference Include="Microsoft.ApplicationInsights.WorkerService" Version="2.22.0"/>
+        <PackageReference Include="Microsoft.Azure.Functions.Worker.ApplicationInsights" Version="1.3.0" />
+        <PackageReference Include="Microsoft.EntityFrameworkCore" Version="8.0.8" />
+        <PackageReference Include="Microsoft.EntityFrameworkCore.Relational" Version="8.0.8" />
+        <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="8.0.8" />
+    </ItemGroup>
+    <ItemGroup>
+        <None Update="host.json">
+            <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+        </None>
+        <None Update="appsettings.json">
+          <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+        </None>
+    </ItemGroup>
+    <ItemGroup>
+        <Using Include="System.Threading.ExecutionContext" Alias="ExecutionContext"/>
+    </ItemGroup>
+    <ItemGroup>
+        <ProjectReference Include="../../../../service/service-directory-api/src/FamilyHubs.ServiceDirectory.Data/FamilyHubs.ServiceDirectory.Data.csproj" />
```

Comment:
Smells like an issue: I don't think we should be referencing the service directory API project from this functions project. If it's just that we need to re-use something in the SD API project, we need to pull it out into a shared package and reference that instead.

Also causes a build error:

```
0>------- Started building project: FamilyHubs.OpenReferral.Function
0>FamilyHubs.OpenReferral.Function.csproj: Error NU1105 : Unable to find project information for '/Users/aaron/src/fh/fh-services/src/service/service-directory-api/src/FamilyHubs.ServiceDirectory.Data/FamilyHubs.ServiceDirectory.Data.csproj'. If you are using Visual Studio, this may be because the project is unloaded or not part of the current solution so run a restore from the command-line. Otherwise, the project file may be invalid or missing targets required for restore.
0>------- Finished building project: FamilyHubs.OpenReferral.Function. Succeeded: False. Errors: 1. Warnings: 0
Build completed in 00:00:00.416
```

Reply:
Thanks for spotting, had this in as a temporary means to build out the project. Moved into the shared kernel and referenced in both projects (SD API & OR Func)

[2cd8f21](https://github.com/DFE-Digital/fh-services/pull/63/commits/2cd8f21a27c5afef2c378063335b921cd0fdb613), [d6ae6eb](https://github.com/DFE-Digital/fh-services/pull/63/commits/d6ae6eb20ab0a7f7220ffb315d9deaafffa1507e) and [bd7db9d](https://github.com/DFE-Digital/fh-services/pull/63/commits/bd7db9dadae3cf6647d9784cedf7da850eb97360)